spec/models/push_notification_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe PushNotification do
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of(:event) }
+  end
+
+  describe 'Relations' do
```

Comment:
I'd rather name it `association` which is the convention from Rails. The way I see it we're are testing it from the AR's perspective.